src/fetchers/ethereum/types.ts

```diff
@@ -1,4 +1,4 @@
-import type { Fetcher } from "../../core"
+import type { Fetcher } from "../../types"
```

Comment:
Good catch! Not sure how it was working before? :thinking:

Reply:
me neither, I was really wondering